bin/oay/DEPENDENCIES.rust.tsv

```diff
@@ -1,289 +0,0 @@
-crate	0BSD	Apache-2.0	Apache-2.0 WITH LLVM-exception	BSD-2-Clause	BSD-3-Clause	BSL-1.0	CC0-1.0	ISC	MIT	MPL-2.0	OpenSSL	Unicode-DFS-2016	Unlicense	Zlib
```

Comment:
Looks strange. Why this file is cleared?

Reply:
Fixed. (My apologies, I overlooked the `[workspace.dependencies]` version in `Cargo.toml`. Since this version doesn't include the patch version, when I searched for the last version, which is 0.44.2, it didn't show up.)
https://github.com/apache/opendal/blob/ddca18b252b2479913966165aabc303f90946fe8/Cargo.toml#L61